modules/UnifiedInterface.py

```diff
@@ -0,0 +1,26 @@
+from modules.configuration.Configuration import Configuration
+from modules.helper.VCenterHelper import VCenterHelper
+from interfaces.vcenter import Vcenter
+from modules.api.Atlas import Atlas
+from modules.helper.EsxiServiceHelper import EsxiServiceHelper
+
+
+class UnifiedInterface:
+
+    def __init__(self, config: Configuration) -> None:
+        self.config = config
+        self.atlas = Atlas(config)
+        self.esxi_helper = EsxiServiceHelper(config)
+        self.vcenter_helper = VCenterHelper(config)
+
+    def get_hosts_by_vcenter(self, vcenter: Vcenter) -> list:
+        return self.netbox.get_hosts_by_vcenter(vcenter)
```

Comment:
Unresolved attribute reference `netbox` for class `UnifiedInterface`.

Reply:
removed that artifact...